addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -1626,94 +1688,113 @@ X[]
                 });
                 it('should not break unbreakables', async () => {
                     await testEditor(BasicEditor, {
-                        contentBefore: `<div class="oe_unbreakable"><div class="oe_unbreakable">` +
-                            `<div class="oe_unbreakable"><div class="oe_unbreakable">[]<br></div>` +
-                            `<div class="oe_unbreakable">abc</div></div></div></div>`,
+                        contentBefore: `<div class="oe_unbreakable">` +
```

Comment:
I don't understand the changes that were done in those unbreakable tests.

Reply:
I just cleaned the content to improve readability.

Those test used to be `<table>` `<tr>` `<td>` to test unbreakable.
Then those tags have been auto replaced by `<div class="oe_unbreakable"></div>`.
But we kept all those unnecessary indentation, so I cleaned them.